### DISCUSSION TITLE
[IntelliSense] Can you specify the license ?

### DISCUSSION THREAD

**Discussion by happysalada** [tailwindlabs/tailwindcss#11941] [Ideas]
what is the license for this project ? MIT , agpl 3.0, bsl ?
I couldn't find any information about the license.

**Comment by wongjn** [comment]
The three packages that make up the VSCode Tailwind Intellisense plugin seem to be MIT, as noted by their LICENSE files:

- [`packages/tailwindcss-language-server/LICENSE`](https://github.com/tailwindlabs/tailwindcss-intellisense/blob/1cc8e62da32895c892624a9f631cd7d16973aa81/packages/tailwindcss-language-server/LICENSE)
- [`packages/tailwindcss-language-service/LICENSE`](https://github.com/tailwindlabs/tailwindcss-intellisense/blob/1cc8e62da32895c892624a9f631cd7d16973aa81/packages/tailwindcss-language-service/LICENSE)
- [`packages/vscode-tailwind/LICENSE`](https://github.com/tailwindlabs/tailwindcss-intellisense/blob/1cc8e62da32895c892624a9f631cd7d16973aa81/packages/vscode-tailwindcss/LICENSE)

**Reply by happysalada** [reply]
Thank you !
